src/annotator-content-analysis.js

```diff
@@ -80,6 +80,19 @@ function annotatorContentAnalysis(options){
 
         })
 
+        $('.commit-question').on('click', (e) => {
```

Comment:
we should avoid coupling our dependency lookup logic with the structure of our dependency model as much as possible. doing it this way dictates that the dependency must be `xx.yy.zz`. i envisioned putting the prev/next buttons on the form itself, setting the dependency as a data-attribute on the answer itself and showing one question at a time, so on click you check $(question).find('input:checked') and asses the dependency from there.